docs/collector-components.md

```diff
@@ -0,0 +1,131 @@
+# Elastic OpenTelemetry Collector components
+
+The OpenTelemetry Collector uses the following components to receive, process, and export telemetry data:
+
+- [Receivers](collector-components.md#receivers): Collect telemetry from your host.
+- [Processors](collector-components.md#processors): Modify or transform telemetry data before sending it to the exporters.
+- [Exporters](collector-components.md#exporters): Send data to the backends or destinations.
+- [Extensions](collector-components.md#extensions): Provide additional functionalities and capabilities.
+
+The default configurations of the Elastic Distribution for the OpenTelemetry Collector follows these flows.
+
+**MacOS and Linux Host metrics:**
+
+```mermaid
+flowchart LR
+    one["`Host metrics receiver`"]
+    two["`Elastic infra metrics processor`"]
+    three["`Attributes processor (dataset)`"]
+    four["`Resource processor (process)`"]
+    five["`Elasticsearch exporter`"]
+    one --> two --> three --> four --> five
+
+    style one fill: #e6f9f7,stroke:#333,stroke-width:1px
+    style two fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style three fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style four fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style five fill:#e6f9f7,stroke:#333,stroke-width:1px
+```
+
+**MacOS and Linux Logs**
+
+```mermaid
+flowchart LR
+    one["`File log receiver`"]
+    two["`Resource detection processor`"]
+    three["`Elasticsearch exporter`"]
+    one --> two --> three
+
+    style one fill: #e6f9f7,stroke:#333,stroke-width:1px
+    style two fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style three fill:#e6f9f7,stroke:#333,stroke-width:1px
+```
+
+**Kubernetes metrics**
+
+```mermaid
+flowchart LR
+    one["`Kubelet stats and host metrics receivers`"]
+    two["`Kubernetes attributes processor`"]
+    three["`Elastic infra metrics processor`"]
+    four["`Resource detection processors (EKS, GCP, K8s)`"]
+    five["`Resource processors (K8s, cloud)`"]
+    six["`Attributes processor (dataset)`"]
+    seven["`Resource processor`"]
+    eight["`Elasticsearch exporter`"]
+    one --> two --> three --> four --> five --> six --> seven --> eight
+
+    style one fill: #e6f9f7,stroke:#333,stroke-width:1px
+    style two fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style three fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style four fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style five fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style six fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style seven fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style eight fill:#e6f9f7,stroke:#333,stroke-width:1px
+```
+
+**Kubernetes logs**

```

Comment:
These arent only Kubernetes logs, the collector also is configured to collect logs from  from a path. So any file there will be collected not just K8s eg. Linux and macOS logs